presto-openapi/src/main/resources/queued_statement.yaml

```diff
@@ -0,0 +1,205 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/QueryResults'
+        '404':
+          description: Query not found
+  /v1/statement/queued/{queryId}/{token}:
+    get:
+      summary: Get the status of a submitted query
+      description: Retrieves the status of a submitted query identified by queryId
+      parameters:
+        - name: queryId
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The ID of the query to retrieve
+        - name: token
+          in: path
+          required: true
+          schema:
+            type: integer
+          description: Monotonically increasing token that identifies the next batch of query results
+        - name: slug
+          in: query
+          required: false
+          schema:
+            type: string
+          description: Unique security token generated for each query that controls access to that query's results
+        - name: maxWait
+          in: query
+          required: false
+          schema:
+            type: string
+            format: duration
+          description: Time to wait for the query to be dispatched
+        - name: binaryResults
+          in: query
+          required: false
+          schema:
+            type: boolean
+          description: Whether to return results in binary format
+        - name: X-Forwarded-Proto
+          in: header
+          required: false
+          schema:
+            type: string
+          description: Forwarded protocol (http or https)
+        - name: Presto-Prefix-URL
+          in: header
+          required: false
+          schema:
+            type: string
+          description: Prefix URL for Presto
+      responses:
+        '200':
+          description: Query status retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref:
+        '404':
+          description: Query not found
+
+    delete:
+      summary: Cancel a query
+      description: Cancels a query identified by queryId
+      parameters:
+        - name: queryId
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The ID of the query to cancel
+        - name: token
+          in: path
+          required: true
+          schema:
+            type: integer
+          description: increasing token that identifies the next batch of query results
+        - name: slug
+          in: query
+          required: false
+          schema:
+            type: string
+          description: Unique security token generated for each query that controls access to that query's results
+      responses:
+        '204':
+          description: Query canceled successfully
+        '404':
+          description: Query not found
+
+
+components:
+  schemas:
+    QueryResults:
```

Comment:
We're missing a lot of fields here. Make sure all the `@JsonProperty`-annotated constructor arguments in `QueryResults.java` are reflected in this schema